Frontend/src/components/Table/Row.svelte

```diff
@@ -3,6 +3,7 @@
   export let item = {};
   export let rowHeight = 40;
   export let rowBackgroundColorFunction;
+  export let cellBackgroundColorFunction;
```

Comment:
```suggestion
  export let cellBackgroundColorFunction = (col, rental) => "";
```
This way we could remove one else block at the bottom.

Reply:
done!